detectsex.rb

```diff
@@ -0,0 +1,80 @@
+require 'telegram/bot'
+require 'net/http'
+require 'json'
+require 'dotenv'
+
+Dotenv.load('.env')
```

Comment:
it's common practice to create .env.example with all needed env variables with fake data and .gitignore with .env and all other files that are secure or unneeded